src/app.js

```diff
@@ -1 +1,55 @@
+/* eslint-disable no-console */
+/* eslint-disable max-len */
 'use strict';
+
+const styles = require('terminal-styles');
+const readline = require('readline');
+
+const randomNumber = require('./getRandomNumber');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const { red, green, x } = styles;
+
+const game = () => {
+  terminal.question('Guess the number: ', (answer) => {
+    if (answer.length !== 4) {
+      console.log(styles`${red}Please enter a number with 4 digits${x}`);
+      game();
+    } else if (answer === randomNumber.toString()) {
+      console.log(styles`${green}You guessed correctly!${x}`);
+      terminal.close();
+    } else {
+      const bulls = answer
+        .split('')
+        .filter((item, index) => item === randomNumber.toString()[index]).length;
```

Comment:
```suggestion
      const bulls = [...answer]
        .filter((item, index) => item === randomNumber[index]).length;
```
String is array-like, so you can spread it directly to new array.